### PR TITLE
Do not set the release file Label or Version during publish

### LINKED ISSUE #443: Release file version changes require manual confirmation in APT
**Version**
Affects all versions of pulp_deb, when used with APT version 1.5 and greater.

**Describe the bug**
pulp_deb publishes the Pulp repo version as the Release file version.
Starting with APT 1.5 and greater, APT asks for user confirmation in order to keep using a repo with a changed Release file version, unless `apt-get update --allow-releaseinfo-change` is used.

This can break automation not using this flag, whenever the Release file version changes.

**Expected behavior**
Do not use the Release file version field in a way that clashes with basic APT design.
We should check if there are other release file fields that require manual confirmation when they change as well!

**Additional context**
See also: https://community.theforeman.org/t/pulp-deb-incorrect-version-in-release-and-inrelease/27416/3

### PR DESCRIPTION
**Pull request by quba42**
Closes #443

This is a quick and dirty workaround to prevent apt-secure from
constantly requiering users to run the following:

apt-get update --allow-releaseinfo-change

This fix changes the pulp_deb publication behaviour. To mitigate, the
old behaviour can be renabled by setting SET_RELEASE_FILE_LABEL and
SET_RELEASE_FILE_VERSION to True.

For a long term fix, additional new features and behaviour changes are
now planned: https://github.com/pulp/pulp_deb/issues/449

### CODE REVIEW

File: CHANGES/443.bugfix [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Changed the release file publication behaviour of the APT publisher to prevent a design clash with apt-secure.
+You may set PUBLISH_RELEASE_FILE_LABEL and PUBLISH_RELEASE_FILE_VERSION to True to revert to the old behaviour.
```
Comment (quba42) [inline CHANGES/443.bugfix:2]: Should I be placing this in a single line? Not sure how towncrier likes it...

**Comment by quba42** [comment]
Successfully tested using the config file values manually.

File: CHANGES/443.bugfix (hunk 1 above)
Comment (mdellweg) [reply CHANGES/443.bugfix:2]: No, you should follow the same coding style as you want to see it land in the Changelog. towncrier will insert it literally (maybe indented...).

Comment (quba42) [reply CHANGES/443.bugfix:2]: Thanks for the tip!

**Pull request merged**